tests/test_trailedSourceFilter.py

```diff
@@ -94,15 +110,43 @@ def test_run_no_trails(self):
         np.testing.assert_array_equal(results.diaSources["diaSourceId"].values, [0, 1, 2, 3, 4])
         np.testing.assert_array_equal(results.trailedDiaSources["diaSourceId"].values, [])
 
+    def test_run_edge(self):
+        """Run trailedSourceFilterTask with the default max distance.
+        filtered out of the final results and put into results.trailedSources.
+        """
+        trailedSourceFilterTask = TrailedSourceFilterTask()
+
+        results = trailedSourceFilterTask.run(self.edgeDiaSources, self.exposure_time)
+
+        self.assertEqual(len(results.diaSources), 3)
+        np.testing.assert_array_equal(results.diaSources['diaSourceId'].values, [0, 2, 3])
+        np.testing.assert_array_equal(results.trailedDiaSources['diaSourceId'].values, [1, 4])
+
     def test_check_dia_source_trail(self):
         """Test the source trail mask filter.
 
         Test that the mask filter returns the expected mask array.
         """
         trailedSourceFilterTask = TrailedSourceFilterTask()
-        mask = trailedSourceFilterTask._check_dia_source_trail(self.diaSources, self.exposure_time)
+        flag_map = os.path.join(utils.getPackageDir("ap_association"), "data/association-flag-map.yaml")
+        unpacker = UnpackApdbFlags(flag_map, "DiaSource")
+        flags = unpacker.unpack(self.diaSources["flags"], "flags")
+        mask = trailedSourceFilterTask._check_dia_source_trail(self.diaSources, self.exposure_time,
+                                                               flags)
```

Comment:
Please consider a different variable name, since "mask" means something pretty specific in Science Pipelines parlance and I'm pretty sure it is not this :)

Reply:
I can swap it to trailed_mask like I have elsewhere, though maybe I should rename the whole lot to not imply they are the different kind of masks...